product_docs/docs/biganimal/release/reference/cli.mdx

```diff
@@ -116,6 +116,7 @@ biganimal config
 | Setting              | Description                                                      |
 | -------------------- | ---------------------------------------------------------------- |
 | `context_credential` | The default credential used in the following commands. |
+| `context_project`    | The default project used to run the following commands. You can specify more than one project using a comma separator.  |
```

Comment:
`You can specify more than one project using a comma separator.` should be removed.